### ISSUE TITLE
Adjust answer in HTML Accessibility Quiz and review page

### ISSUE THREAD

**Issue by kaipepler** [freeCodeCamp/freeCodeCamp#59808]
### Describe the Issue

In the HTML Accessibility Quiz, one of the questions is as follows:

**Q:** What is the role of the `aria-hidden` attribute?
**A:** This attribute is used to hide an element from people using assistive technologies.

I suggest adjusting the wording of this answer, because there's an opportunity for more precise language:

`ARIA-hidden` isn't meant to hide content from the **people** using assistive technologies, it's meant to hide content from the Accessibility API, thus removing it from the **assistive technologies** themselves. I know this is a bit of a pedantic change, but one I think would be helpful for those new to accessibility, because it doesn't make assumptions about the wide variety of people who use assistive technology. Many screen reader users have some amount of sight, so the content wouldn't necessarily be hidden from them, it would just not be read via their screen reader.

I opened PR #59799 and was recommended that I open an issue to discuss.

Thank you!

### Affected Page

https://www.freecodecamp.org/learn/full-stack-developer/quiz-html-accessibility/quiz-html-accessibility

### Your code

```



```

### Expected behavior

Adjust the language of the answer so the focus is on how ARIA affects assistive technologies. For example:

Q: What is the role of the aria-hidden attribute?
A: This attribute is used to hide an element from assistive technologies.

### System

- Device: Laptop
- OS: macOS 15.4.1
- Browser: Safari
- Version: 18.4

**Comment by ilenia-magoni** [comment]
the same sentence is used in the review: https://www.freecodecamp.org/learn/full-stack-developer/review-html-accessibility/review-html-accessibility

and probably in the lectues

if the change is needed, it needs to be applied in multiple places

**Comment by ilenia-magoni** [comment]
the same change to the accessiblity review should be done to the global html review https://www.freecodecamp.org/learn/full-stack-developer/review-html/review-html
